latest/apidoc/manager/font/Droid_Sans/LICENSE.txt

```diff
@@ -1,202 +1,202 @@
-
```

Comment:
What's going on with these license files? Are they just indentation changes or something? If so, I think it's better just to leave that out.

Reply:
Or is it autogenerated? In which case it probably doesn't matter too much.

Reply:
I am not sure, I didn't actively changed the indentation of this files. I will push a fixup and remove these.